packages/sirius-web/frontend/sirius-web/src/reset.css

```diff
@@ -201,7 +201,6 @@ button {
   border: 0;
   color: inherit;
   font: inherit;
-  line-height: normal;
```

Comment:
Honesty I think the fix should be in the CSS of the Reactflow label. You can't just assume that every single application in which our diagrams will be used will fallback to some expected CSS value.

Reply:
I'll do that.